### PR TITLE
New buttons

### PR DESCRIPTION
**Pull request by tomgenoni**
New button styles from Linda. Attached is a screenshot, live buttons here: http://optimizely.github.io/proto/buttons/

screen shot 2015-09-23 at 3 40 16 pm

### CODE REVIEW

**Comment by tomgenoni** [comment]
@danoc @kwalker3690 please review.

**Comment by kwalker3690** [comment]
Its fine.

File: src/core/partials/objects/_buttons.scss [hunk 1]
```diff
@@ -54,111 +57,129 @@
...
     &:hover,
     &:active,
-    &:visited,
     &:focus {
-      color: map-fetch($color, text white) !important;
+      color: map-fetch($color, text white);
     }
   }
 
-  &--dismiss {
```
Comment (danoc) [inline src/core/partials/objects/_buttons.scss]: This change makes it a breaking change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 ## [Unreleased][unreleased]
 ### Added
 - [Patch] Adding `watch` process so that Sass changes will compile automatically. To use run `gulp watch`.
+- [Patch] Adding `_buttons--deprecated.scss` to temporarily retain the `button--dismiss` styling.
+
+## Changed
+- [Patch] Removed unneeded `!important` on colors by using `:visited` to fix conflicts with anchors styled as buttons.
+- [Patch] Updating buttons to use new styling, `button--dismiss` is now `button--plain`.
```
Comment (danoc) [inline CHANGELOG.md]: There were some removed classes. Can you note these and make it a release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 ## [Unreleased][unreleased]
 ### Added
 - [Patch] Adding `watch` process so that Sass changes will compile automatically. To use run `gulp watch`.
+- [Patch] Adding `_buttons--deprecated.scss` to temporarily retain the `button--dismiss` styling.
```
Comment (danoc) [inline CHANGELOG.md]: This should be under:

```
### Deprecated
```

**Comment by danoc** [comment]
needs fixing.

**Comment by danoc** [comment]
lgtm!

**Pull request merged**